examples/js/vr/WebVR.js

```diff
@@ -131,7 +131,7 @@ var WEBVR = {
 
 		}
 
-		if ( 'xr' in navigator ) {
+		if ( 'xr' in navigator && 'requestDevice' in navigator.xr ) {
```

Comment:
Should we update `WebGLRenderer`, too? https://github.com/mrdoob/three.js/blob/dev/src/renderers/WebGLRenderer.js#L311

Reply:
Yes. I think it makes sense to use the same test at both places.